layouts/download_list.ejs

```diff
@@ -0,0 +1,42 @@
+<% for (const item in options) { %>
+  <% if (options[item].hasOwnProperty("url")) { %>
+  <li class="list-item">
+    <div class="list-item__inner">
+      <div class="list-item__icon"></div>
+      <<%=heading_lvl%> class="list-item__name">
+        <a href="<%=options[item].url%>"><%=options[item].title%></a>
+      </<%=heading_lvl%>>
+      <% if (options[item].hasOwnProperty("alerts")) { %>
+        <% for (const alert of options[item].alerts) { %>
+          <div class="list-item__alert icon-alert">
+            <%=alert%>
+          </div>
+        <% } %>
+      <% } %>
+    </div>
+  </li>
+  <% } else { %>
+  <li class="list-item list-item__dropdown">
+    <div class="list-item__inner">
+      <div class="list-item__icon"></div>
+      <<%=heading_lvl%> class="list-item__name"><%= options[item].title %></<%=heading_lvl%>>
+      <div class="list-item__summary"><%= options[item].description %></div>
+
+      <% if (options[item].hasOwnProperty("alerts")) { %>
+        <% for (const alert of options[item].alerts) { %>
+          <div class="list-item__alert icon-alert">
+            <%=alert%>
+          </div>
+        <% } %>
+      <% } %>
+
+      <ul class="list-item__expandable hide" id="expandable-<%= item_key.concat("-",item) %>" data-status="closed">
+        <%- include("download_list", { options: options[item].options, item_key: item_key.concat("-",item), heading_lvl: "h2" }) %>
+      </ul>
+    </div>
+    <div class="list-item__dropdown-btn">
+      <button id="expandable-<%= item_key.concat("-",item) %>-btn" class="icon-chevron-down" onclick="expandOrCollapseDownloadSection('expandable-<%=item_key.concat("-",item)%>-btn')"></button>
+    </div>
```

Comment:
Would you consider using a [`details`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/details) element for this?

Also, the expand button doesn't have an [accessible name](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name).

Reply:
Thanks a ton for your feedback on this! Sorry it's taken so long for me to see it.
This section has been largely reworked by @savetheclocktower and hopefully that address the accessibility concerns. Appreciate your help!